### ISSUE TITLE
Get load balancer target group

### ISSUE THREAD

**Issue by ruciu** [aws/elastic-beanstalk-roadmap#135]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
* Please do not leave "+1" or "me too" comments. They generate extra noise for issue followers and do not help prioritize the request.

**Tell us about your request**
I would like to be able to get access to LB target group created by Elastic Beanstalk environment

**Is this request specific to an Elastic Beanstalk [platform](https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platforms-supported.html)?**
No

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
I would like to be able to pass target group name in my Terraform script to use it with shared load balancer.

**Are you currently working around this issue?**
I had to check it in console and hardcode it

**Additional context**
None of CreateEnvironment(https://docs.aws.amazon.com/elasticbeanstalk/latest/api/API_CreateEnvironment.html) and DescribeEnvironment(https://docs.aws.amazon.com/elasticbeanstalk/latest/api/API_DescribeEnvironments.html) makes it possible now. If target group would be searchable by other means(e.g tags) it would be fine too.

**Comment by pprahlad** [comment]
Can you not get this through Describe Target Group? https://docs.aws.amazon.com/cli/latest/reference/elbv2/describe-target-groups.html

**Comment by mihajlovics** [comment]
@pprahlad No, that is not supported. We can't automate Cloudwatch target group monitoring since this is not supported.

**Comment by danieljamesscott** [comment]
> @pprahlad No, that is not supported. We can't automate Cloudwatch target group monitoring since this is not supported.

It's possible to do this in terraform with a data lookup using the resource tagging API - it's a bit nasty, but it does work.
